sycl/include/CL/sycl/handler2.hpp

```diff
@@ -76,20 +76,20 @@ namespace detail {
                                                                                \
   template <class DstT> struct InitSizesST##POSTFIX<1, DstT> {                 \
     static void initSize(DstT &Dst) {                                          \
-      Dst[0] = cl::__spirv::get##POSTFIX<0>();                                 \
+      Dst[0] = get##POSTFIX<0>();                                 \
```

Comment:
Also not sure it matters too much, but I think the macro "\" indentation is off here. Otherwise excellent patch, getting rid of 15k loc~!

Reply:
Ah I commented before reading the merge, can disregard the comments as they're pretty inane anyway!